.github/pull_request_template.md

```diff
@@ -1,18 +1,23 @@
 This PR changes / adds / introduces the feature of ....
 
+Acceptance Criteria:
+- [ ] first criteria (maybe a test case)
+- [ ] second criteria
+- [ ] third citeria
+
+
 Testing steps:
 - first step
 - second step
 - ....
 
-Acceptance Criteria:
-- first criteria (maybe a test case)
-- second criteria
-- third citeria
-
-Solves issue #1.
-
 Steps needed for migration:
 - first migration step
 - second migration step
 - ...
+
+---
+
```

Comment:
Could you please also add 
```suggestion

- resolves issue #number

```
` - resolves issue #number`
here?

Reply:
Thanks I don't know why, but this Line was actually inside, actually it was the header of the whole PR. I'm going to fix this.